region/maridia/inner-yellow/Bug Sand Hole.json

```diff
@@ -469,8 +456,42 @@
       "id": 20,
       "link": [1, 4],
       "name": "Base",
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": []
+    },
+    {
+      "link": [1, 4],
+      "name": "Fall Into Room With Mockball",
+      "entranceCondition": {
+        "comeInWithMockball": {
+          "adjacentMinTiles": 0,
+          "remoteAndLandingMinTiles": [[0, 0]],
+          "speedBooster": "any"
+        }
+      },
       "requires": []
     },
+    {
+      "link": [1, 5],
+      "name": "Jump Through Room",
+      "entranceCondition": {
+        "comeInJumping": {
+          "speedBooster": "any",
+          "minTiles": 1
+        }
+      },
+      "requires": [
+        "canPrepareForNextRoom",
+        "canTrickyJump",
+        "canPlayInSand"
+      ],
+      "note": [
+        "Jump through the door to cross most of the room, then jump off of the sand to cross the rest.",
+        "Crossing the room with enough speed to cross with one jump will set Samus up to be grabbed by the Yapping Maw."
```

Comment:
```suggestion
        "With enough speed, crossing the whole room without landing in the sand is also possible, but Samus must still jump quickly after landing to avoid being grabbed by the Yapping Maw."
```